src/metric/definition.go

```diff
@@ -405,6 +405,12 @@ var EtcdSpecs = definition.SpecGroups{
 				ValueFunc: prometheus.FromValueWithOverriddenName("go_goroutines", "goGoroutines"),
 				Type:      sdkMetric.GAUGE,
 			},
+			// computed
```

Comment:
is it any difference between `computed` and `calculated` metrics?

Reply:
no, it's probably I just that I did it at different times